source/_patterns/03-organisms/referrer-destination-renderer.twig

```diff
@@ -0,0 +1,22 @@
+<input id="pattern_lib_mock_marker" type="hidden" value="true"/>
+
+{% include "molecules-referrer-destination-form" with {
```

Comment:
Patternlib supports json files for test data (both globally and per pattern file). So I move the test model for this to a separate .json file.